cms/tests/test_views.py

```diff
@@ -51,8 +51,9 @@ def setUp(self):
         })
 
         languages = ("[('eng_GB', 'English'), ('swa_KE', 'Swahili'),"
-                     "('spa_ES', 'Spanish'), ('fra_FR', 'French'),"
-                     "('hin_IN', 'Hindi'), ('ind_ID', 'Bahasa')]")
+                     "('spa_ES', 'Spanish'), ('fre_FR', 'French'),"
```

Comment:
No, we chose the `iso639-2/B` and we should stick to it. We only need `iso639-2/T` for displaying the locale only. The only change I made there was `fra_FR` - I changed it yesterday to make the tests pass.